src/Request.php

```diff
@@ -10,6 +10,7 @@
 
 namespace Longman\TelegramBot;
 
+use Longman\TelegramBot\TelegramBot;
```

Comment:
I assume you mean `use Longman\TelegramBot\TelegramLog` here, right?
If so, you can remove this line completely, as `TelegramLog` is on the same namespace level.